### PR TITLE
Minor CLI Fix for Formatting Tests

### LINKED ISSUE #320: CLI does not format tests

### PR DESCRIPTION
**Pull request by alderwhiteford**
Feel free to close this PR if you don't want the formatter on tests for some reason

### CODE REVIEW

**Review by garrettladley — CHANGES_REQUESTED** [review]
This might have unintended consequences that cause a regression in other commands. Try splitting into BACKEND_SRC and BACKEND vars to add this feature while maintaining parity

**Comment by alderwhiteford** [comment]
> This might have unintended consequences that cause a regression in other commands. Try splitting into BACKEND_SRC and BACKEND vars to add this feature while maintaining parity

true true

File: cli/commands/config.go [hunk 1]
```diff
@@ -8,9 +8,10 @@ import (
...
 var (
-	ROOT_DIR, _    = utils.GetRootDir()
-	FRONTEND_DIR   = filepath.Join(ROOT_DIR, "/frontend")
-	BACKEND_DIR    = filepath.Join(ROOT_DIR, "/backend/src")
-	CONFIG, _      = config.GetConfiguration(filepath.Join(ROOT_DIR, "/config"), false)
-	MIGRATION_FILE = filepath.Join(BACKEND_DIR, "/migrations/data.sql")
+	ROOT_DIR, _     = utils.GetRootDir()
+	FRONTEND_DIR    = filepath.Join(ROOT_DIR, "/frontend")
+	BACKEND_DIR     = filepath.Join(ROOT_DIR, "/backend/")
+	BACKEND_SRC_DIR = filepath.Join(ROOT_DIR, "/backend/src")
```
Comment (garrettladley) [inline cli/commands/config.go:14]:
```suggestion
	BACKEND_SRC_DIR = filepath.Join(BACKEND_DIR, "/src")
```

File: cli/commands/config.go [hunk 2]
```diff
@@ -8,9 +8,10 @@ import (
...
 
 var (
-	ROOT_DIR, _    = utils.GetRootDir()
-	FRONTEND_DIR   = filepath.Join(ROOT_DIR, "/frontend")
-	BACKEND_DIR    = filepath.Join(ROOT_DIR, "/backend/src")
-	CONFIG, _      = config.GetConfiguration(filepath.Join(ROOT_DIR, "/config"), false)
-	MIGRATION_FILE = filepath.Join(BACKEND_DIR, "/migrations/data.sql")
+	ROOT_DIR, _     = utils.GetRootDir()
+	FRONTEND_DIR    = filepath.Join(ROOT_DIR, "/frontend")
+	BACKEND_DIR     = filepath.Join(ROOT_DIR, "/backend/")
```
Comment (garrettladley) [inline cli/commands/config.go:13]:
```suggestion
	BACKEND_DIR     = filepath.Join(ROOT_DIR, "/backend")
```
nit for consistency

**Review by garrettladley — CHANGES_REQUESTED** [review]

**Pull request merged**